pkg/kubectl/cmd/util/printing.go

```diff
@@ -82,10 +85,54 @@ func ValidateOutputArgs(cmd *cobra.Command) error {
 	return nil
 }
 
-// printerForOptions returns the printer for the outputOptions (if given) or
-// returns the default printer for the command. Requires that printer flags have
-// been added to cmd (see AddPrinterFlags).
-func printerForOptions(options *printers.PrintOptions) (printers.ResourcePrinter, error) {
+// PrintSuccess prints a success message and can do a "-o name" as "shortOutput"
+// TODO this should really just a printer
```

Comment:
?

Reply:
I think this TODO goes along the other one, when we have `PrintOptions` it'll be handled by a single printer.

Reply:
> ?

Updated.  Printing success for an object is just a particular kind of "print my object" which is exactly what our Printer signatures are for.  This method should die and be replaced by a proper "successprinter"